src/app/containers/ConsentBanner/useConsentBanners/getCookieOvenEndpoints.js

```diff
@@ -0,0 +1,21 @@
+export default ({ origin, policy }) => {
```

Comment:
This file used to be called cookieOvenUrl.js. Check the diffs here:

https://www.diffchecker.com/cmUgkrVu

Reply:
The diff links have expired 🙁